eng/build-utils.ps1

```diff
@@ -333,6 +333,12 @@ function Make-BootstrapBuild([switch]$force32 = $false) {
   Write-Host "Cleaning Bootstrap compiler artifacts"
   Run-MSBuild $projectPath "/t:Clean" -logFileName "BootstrapClean"
 
+  # Work around NuGet bug that doesn't correctly re-generate our project.assets.json files.
```

Comment:
I am assuming that just finding and deleting those project.assets.json files isn't going to give the result we want?